app/views/mediatorConfigModal.html

```diff
@@ -0,0 +1,48 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="cancel()" aria-hidden="true">&times;</button>
+  <h4 class="modal-title" id="addMediatorConfigLabel">
+    <i class="glyphicon glyphicon-th"></i>&nbsp;&nbsp;
+    Edit mediator config
+  </h4>
+</div>
+<form name="form" novalidate>
+  <div class="modal-body">
+    <fieldset>
+
+      <div class="form-group-2columns" ng-repeat="def in mediator.configDefs">
+        <div class="form-group" style="float: left;">
+          <label>{{def.param}}</label>
+        </div>
+        <div class="form-group" style="float: right;">
+
+          <div ng-if="def.type === 'string'">
+            <input class="form-control" type="text" ng-model="mediator.config[def.param]" ng-disabled="update">
```

Comment:
So currently in core we have a `def.default` property (which isn't be used here) and the mediator can supply it own default config in the `config` property. I now think this is duplicative. Perhaps we should remove the `def.default` property from core and just use the defaults provided in the `config` property. What do you think?